serving/src/config.rs

```diff
@@ -78,59 +80,9 @@ impl Settings {
 #[cfg(test)]
 mod tests {
     use std::env;
-    use std::fs::File;
-    use std::io::Write;
-
-    use tempfile::tempdir;
 
     use super::*;
 
-    #[tokio::test]
-    async fn test_settings_load() {
```

Comment:
@yhl25  is this a valid test?

Reply:
we don't need this, since we never create the Settings this way. `test_config` should test the config part.